splunk_eventgen/eventgen_core.py

```diff
@@ -240,17 +240,22 @@ def _create_generator_workers(self, workercount=20):
             self.workerPool = []
             for worker in range(workercount):
                 # builds a list of tuples to use the map function
+                disable_logging = True if self.args and self.args.disable_logging else False
```

Comment:
if we set the default value in arg parser, no need to use if else here. Just use
`disable_logging = self.args.disable_logging`

Reply:
`self.args` might be `None` here.